sechub-doc/src/test/java/com/daimler/sechub/restdoc/ProjectAdministrationRestControllerRestDocTest.java

```diff
@@ -273,19 +280,86 @@ public void restdoc_show_project_details() throws Exception {
 						),
 				responseFields(
-							fieldWithPath(ProjectDetailInformation.PROPERTY_PROJECT_ID).description("The name of the project"),
-							fieldWithPath(ProjectDetailInformation.PROPERTY_USERS).description("A list of all users having access to the project"),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_PROJECT_ID).description("The name of the project."),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_USERS).description("A list of all users having access to the project."),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_OWNER).description("Username of the owner ofthis project. An owner is the person in charge."),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_WHITELIST).description("A list of all whitelisted URIs. Only these ones can be scanned for the project!"),
 							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA).description("An JSON object containing metadata key-value pairs defined for this project."),
-							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA + ".key1").description("An arbitrary metadata key")
+							fieldWithPath(ProjectDetailInformation.PROPERTY_METADATA + ".key1").description("An arbitrary metadata key."),
+							fieldWithPath(ProjectDetailInformation.PROPERTY_DESCRIPTION).description("The project description.")
 						)
 					)
 				);
 
 		/* @formatter:on */
     }
 
+    @Test
+    @UseCaseRestDoc(useCase = UseCaseAdministratorChangesProjectDetails.class)
+    public void restdoc_change_project_description() throws Exception {
+        /* prepare */
+        Project project = mock(Project.class);
+        when(project.getId()).thenReturn("projectId1");
+
+        Set<User> users = new LinkedHashSet<>();
+        User user1 = mock(User.class);
+        when(user1.getName()).thenReturn("name1");
+
+        User user2 = mock(User.class);
+        when(user2.getName()).thenReturn("name2");
+
+        users.add(user1);
+        users.add(user2);
+
+        when(project.getUsers()).thenReturn(users);
+        when(project.getOwner()).thenReturn(user1);
+        Set<URI> whiteList = new LinkedHashSet<>();
+        whiteList.add(new URI("http://www.sechub.example.org"));
+        when(project.getWhiteList()).thenReturn(whiteList);
+
+        Set<ProjectMetaDataEntity> metaData = new LinkedHashSet<>();
+        ProjectMetaDataEntity entry = new ProjectMetaDataEntity("projectId1", "key1", "value1");
+        metaData.add(entry);
+
+        when(project.getMetaData()).thenReturn(metaData);
+        
+        when(project.getDescription()).thenReturn("description");
+
+        ProjectDetailInformation detailInformation = new ProjectDetailInformation(project);
+        
+        when(detailsChangeService.changeDetails(any(), any())).thenReturn(detailInformation);
+
+        /* execute + test @formatter:off */
+        this.mockMvc.perform(
+                put(https(PORT_USED).buildAdminChangesProjectDescriptionUrl(PROJECT_ID.pathElement()), "projectId1").
```

Comment:
`put` is not the correct verb here.

Reply:
see my other comment on `PATCH` vs `PUT`.